ppr-ui/src/components/common/ContactInformation.vue

```diff
@@ -18,41 +18,46 @@
       class="mt-8 pa-8 pr-6"
       :class="{ 'border-error-left': showBorderError }"
     >
-      <v-row no-gutters>
-        <v-col cols="12" sm="3">
+      <v-row no-gutters justify="space-between">
+        <v-col cols="12" sm="2" class="mt-1">
           <label
             class="generic-label"
             :class="{ 'error-text': showBorderError }"
           >
             {{ content.sideLabel }}
           </label>
         </v-col>
-        <v-col cols="12" sm="9" class="px-1">
+        <v-col cols="12" sm="10" class="px-1">
           <v-radio-group
             id="contact-info-type-options"
             v-model="contactInfoType"
             class="mt-0 pr-1" row
             hide-details="true"
-            @change="clearFields($event)"
           >
             <v-radio
               id="person-option"
               class="person-radio"
               label="Individual Person"
               active-class="selected-radio"
-              :value="SubmittingPartyTypes.PERSON"
+              :value="ContactTypes.PERSON"
             />
             <v-radio
               id="business-option"
               class="business-radio"
               label="Business"
               active-class="selected-radio"
-              :value="SubmittingPartyTypes.BUSINESS"
+              :value="ContactTypes.BUSINESS"
             />
           </v-radio-group>
 
           <v-divider class="my-9 ml-0 mr-2" />
 
+          <CautionBox v-if="contactInfoModel.hasUsedPartyLookup"
+           class="mb-9"
+           set-msg="If you make changes to the submitting party information below, the changes will
```

Comment:
This message can be made more dynamic in the future but we are currently only using it for submitting party

Reply:
Although it works both ways, we general go with camelCase on the props

Reply:
whoops missed that. My code editor always auto completes to lower case with a dash on the props